blankly/utils/utils.py

```diff
@@ -468,8 +468,12 @@ def get_ohlcv(candles, n):
     new_candles['volume'] = sum_period(candles['volume'].astype(float), n, True)
```

Comment:
Can you add docs to this or more descriptive arguments? I'm not sure exactly what everything does.